packages/components/inputs/search-select-input/src/search-select-input.js

```diff
@@ -0,0 +1,151 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { ThemeContext } from '@emotion/core';
+import { SafeHTMLElement } from '@commercetools-uikit/utils';
+import { SearchIconDropdownIndicator } from '@commercetools-uikit/select-utils';
+import AsyncSelectInput from '@commercetools-uikit/async-select-input';
+import messages from '../../../../../src/components/internals/messages/select';
+import createSelectStyles from '../../../../../src/components/internals/create-select-styles';
+
+const useTheme = () => useContext(ThemeContext);
+
+const SearchSelectInput = (props) => {
+  const intl = useIntl();
+  const theme = useTheme();
+  const noOptionsMessageWithOutInputValue = props.initialSuggestionMessage
```

Comment:
Just to keep the camelCasing consistent 😁 

```suggestion
  const noOptionsMessageWithoutInputValue = props.initialSuggestionMessage
```